pgx-macros/src/lib.rs

```diff
@@ -937,10 +940,19 @@ enum DogNames {
 ```
 
 */
-#[proc_macro_derive(PostgresOrd)]
+#[proc_macro_derive(PostgresOrd, attributes(pgx))]
 pub fn postgres_ord(input: TokenStream) -> TokenStream {
     let ast = parse_macro_input!(input as syn::DeriveInput);
-    impl_postgres_ord(ast).into()
+    let to_sql_config = match sql_entity_graph::ToSqlConfig::from_attributes(ast.attrs.as_slice()) {
```

Comment:
Do you think we could move this into the `impl_postgres_ord` function? I think it'd be more testable. (I do aspire to get these well tested one day...)